crates/primitives/starknet/src/transaction/mod.rs

```diff
@@ -429,7 +429,7 @@ impl Transaction {
         };
 
         // FIXME 710
-        let mut initial_gas = super::constants::INITIAL_GAS;
+        let mut initial_gas = self.max_fee.try_into().unwrap(); // unwrap safe as >u64 case will be handled at client side
```

Comment:
shouldn't it be independent of the client?

Reply:
what should be independent of the client?

Reply:
the safeness (or not) of the unwrap

Reply:
makes sense, fixed